linkerd/app/src/env.rs

```diff
@@ -1005,6 +1021,7 @@ impl fmt::Display for EnvError {
         match self {
             EnvError::InvalidEnvVar => write!(f, "invalid environment variable"),
             EnvError::NoDestinationAddress => write!(f, "no destination service configured"),
+            EnvError::ConflictingConfig => write!(f, "conflicting configuration"),
```

Comment:
FWIW, I believe we just handle this elsewhere as `InvalidEnvVar`. I'd probably be inclined to leave it that way, otherwise we should probably go update these other cases to use the new error variant.